lib/Chunk.js

```diff
@@ -438,9 +438,7 @@ class Chunk {
 	 * @returns {boolean} returns true if chunk does exist in chunkGroup and is removed
 	 */
 	removeGroup(chunkGroup) {
```

Comment:
addGroup above need to be changed in similar way.

Reply:
Unfortunately, it works other way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/add#Return_value

Reply:
We can change to `add`, `has` chain, don't know will this impact performance.

Reply:
ah I see what you did.

You can remove the return values from `removeGroup` and `addGroup` and make them return void methods. The result is never used in the codebase.

Reply:
Done